style.css

```diff
@@ -0,0 +1,67 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+section{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    min-height: 100vh;
+    background: #042104;
+
+    animation: animateBg 10s linear infinite;
+}
+@keyframes animateBg{
+    0%{
+        filter: hue-rotate(0deg);
+    }
+    100%{
+        filter: hue-rotate(360deg);
+    }
+}
+section .loader{
+    position: relative;
+    width: 120px;
+    height: 120px;
+}
+section .loader span{
+    position: absolute;
+    top: 0;
+    left: 0;
+
+    width: 100%;
+    height: 100%;
+
+    transform: rotate(calc(18deg * var(--i)));
+}
+section .loader span:before{
```

Comment:
see this will be exactly what i was trying to say, use the concepts of specificity and pseudo classes they will reduce the size of your selector and make it more readable

Reply:
done!